cristin-project/src/test/java/no/unit/nva/cristin/projects/query/organization/QueryCristinOrganizationProjectHandlerTest.java

```diff
@@ -131,6 +136,44 @@ void shouldReturnSearchResponseWithEmptyHitsWhenBackendFetchIsEmpty() throws Api
         assertThat(0, equalTo(searchResponse.getHits().size()));
     }
 
+    @Test
+    void shouldAddParamsToCristinQueryForFilteringAndReturnOk() throws IOException, ApiGatewayException {
+        cristinApiClient = spy(cristinApiClient);
+        doReturn(new HttpResponseFaker(EMPTY_LIST_STRING,
+                HttpURLConnection.HTTP_OK, generateHeaders(ZERO_VALUE, LINK_EXAMPLE_VALUE)))
+                .when(cristinApiClient).listProjects(any());
+        handler = new QueryCristinOrganizationProjectHandler(cristinApiClient, new Environment());
+        var queryParams = Map.of("funding", "NRE:1234",
```

Comment:
I was advised to use strings as constant variables when started working in the project, but maybe you can reuse the names you are using in the query parameter map where you defined a lot of constants, instead of writing them again. Also, we have a library generating random data is I have described in other comment. I don't think we need exact matches for all query params since we are mocking upstream and just validating that the service accepts the params and puts them on URI